src/event/entity/EntityDamageEvent.php

```diff
@@ -161,7 +161,6 @@ public function canBeReducedByArmor() : bool{
 			case self::CAUSE_VOID:
 			case self::CAUSE_MAGIC:
 			case self::CAUSE_SUICIDE:
-			case self::CAUSE_FALLING_BLOCK:
```

Comment:
This is incorrect. It will allow all armour pieces to reduce the damage.

Reply:
> This is incorrect. It will allow all armour pieces to reduce the damage.

This is correct, This appears to match Bedrock as of 1.19.51.

Reply:
Hmm, you're right. That's pretty stupid ...